src/ledger/LedgerTxnOfferSQL.cpp

```diff
@@ -895,4 +675,131 @@ LedgerTxnRoot::Impl::bulkLoadOffers(
     return populateLoadedEntries(keys,
                                  mDatabase.doDatabaseTypeSpecificOperation(op));
 }
+
+static void
+processAsset(Asset& asset, AssetType assetType, std::string const& issuerStr,
```

Comment:
It might be nice to name this and the loadAllOffers helper below in a way that indicates they only exist for the sake of migration.

Reply:
Agree with this, I scratched my head a few times before realizing that this was migration code